.markdownlint.yaml

```diff
@@ -1,4 +1,5 @@
 MD001: false
 MD013: false
 MD014: false
-MD024: false
\ No newline at end of file
+MD024: false
+MD033: false # Allow inline HTML
```

Comment:
Inline HTML is used all over the place so it doesn't make sense to have a linter rule against it.